### PR TITLE
Added MVP specs

### PR DESCRIPTION
**Pull request by joicemjoseph**
MVP specs and some other files are added.

### CODE REVIEW

File: architecture/minimum_viable_product.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+In my idea, The minimum viable product is as follows
+
+## Server
+Backend server should be capable of recieving a request with current location from end user. Response of the request is list of auto rickshaws around current location. 
+
+Another facility needed is  registering auto rickshaw cooperative unions and auto rickshaw drivers.
+Drivers will register under a cooperative union and cooperative union will verify the driver.
+Driver rating is another capability needed for MVP.
+
+In some areas, where fare meters is not practical, like villages, it is not possible.
```
Comment (rashivkp) [inline architecture/minimum_viable_product.md:12]: IMHO , we can stop considering the village drivers. that creates confusions in my mind :smile:

File: architecture/minimum_viable_product.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Minimum Viable Product.
+The mvp should be so simple as it should be. There are so many alternatives.
+In my idea, The minimum viable product is as follows
+
+## Server
+Backend server should be capable of recieving a request with current location from end user. Response of the request is list of auto rickshaws around current location. 
+
+Another facility needed is  registering auto rickshaw cooperative unions and auto rickshaw drivers.
+Drivers will register under a cooperative union and cooperative union will verify the driver.
+Driver rating is another capability needed for MVP.
```
Comment (rashivkp) [inline architecture/minimum_viable_product.md:10]: is the cooperative union registration is necessary ? 
as I understood, the cooperative union is hosting the services. they may have admin login, which can be used to do some privileged actions. such as blocking a user, driver.. etc

File: architecture/minimum_viable_product.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Mobile End User Client
+The Android mobile client should be capable of
+* requesting nearby rickshaws with user's current location 
+* process its response and display list of auto rickshaws available.
+* Detailed view of a single auto rickshaw will have the following
+    * Auto Rickshaw Photo
+    * Driver Name
+    * Driver phone number for calling.
+    * facilities availiable such as music, fare meter etc.
```
Comment (rashivkp) [inline architecture/minimum_viable_product.md:26]: fare meter is mandatory. right ?

File: architecture/minimum_viable_product.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+Another facility needed is  registering auto rickshaw cooperative unions and auto rickshaw drivers.
+Drivers will register under a cooperative union and cooperative union will verify the driver.
+Driver rating is another capability needed for MVP.
+
+In some areas, where fare meters is not practical, like villages, it is not possible. 
+There, a commonly agreed fare by the drivers cooperative society is getting paid.
+We can use that and give available to the users. 
+
+People who dont have a smart phone could be able to register in the server, manually by the cooperative society with the basic details.
```
Comment (rashivkp) [inline architecture/minimum_viable_product.md:16]: we can exclude this from mvp.

File: architecture/minimum_viable_product.md (hunk 2 above)
Comment (joicemjoseph) [inline architecture/minimum_viable_product.md:10]: I mentioned Cooperative union as a group, privileged for handling drivers and rides in the specified territory.
we haven’t implemented distributed computation. So there will be a centralized server. So the coop is not hosting the service. they receive service from LibreCoop India

File: architecture/minimum_viable_product.md (hunk 3 above)
Comment (joicemjoseph) [inline architecture/minimum_viable_product.md:26]: we cant make this mandatory. but we can introduce as a feature. a feature for filtering in search. we can use the commonly agreed fare chart in the future for automation purpose.

File: architecture/minimum_viable_product.md (hunk 4 above)
Comment (joicemjoseph) [inline architecture/minimum_viable_product.md:16]: which one ? line number 16 ?
if you are mentioning line no 16, then let me tell you that, we can include people who don't have smart phone to our system.
their auto will also be listed and so they will get a ride request.

**Pull request merged**